pkg/kubelet/kubelet.go

```diff
@@ -744,7 +748,7 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 
 	// setup volumeManager
 	klet.volumeManager = volumemanager.NewVolumeManager(
-		kubeCfg.EnableControllerAttachDetach,
+		klet.enableControllerAttachDetach,
```

Comment:
nit: prefer using kubeCfg to the variable in the kubelet struct.  I believe this variable (enableControllerAttachDetach) doesn't need to be stored in the kubelet struct.